### PR TITLE
update xccdf variable options for auditd and audispd

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add / remove options for xccdf variables

#### Rationale:

Options were no talligned with Audit documentation - some were missing, some variables included invalid options.

### CODE REVIEW

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_error_action.var [hunk 1]
```diff
@@ -10,8 +10,9 @@ interactive: false
 
 options:
     default: single
-    email: email
     exec: exec
     halt: halt
     single: single
+    suspend: suspend
```
Comment (jan-cerny) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_error_action.var:16]: according to `man auditd.conf` there can also be `ignore`

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var [hunk 2]
```diff
@@ -10,8 +10,9 @@ interactive: false
 
 options:
     default: single
-    email: email
+
     exec: exec
     halt: halt
     single: single
+    suspend: suspend
```
Comment (jan-cerny) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var:17]: according to man page there is also `ignore` and `rotate`

**Comment by vojtapolasek** [comment]
Do we want to allow "ignore" variable at all?

**Comment by redhatrises** [comment]
> Do we want to allow "ignore" variable at all?

Great point @vojtapolasek . Use of "ignore" isn't a good security option, and IIRC is was removed and not supported for that reason.

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var [hunk 3]
```diff
@@ -10,8 +10,9 @@ interactive: false
 
 options:
     default: single
-    email: email
+
```
Comment (redhatrises) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var:13]: Why is email removed? Also added empty space here.

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_error_action.var [hunk 4]
```diff
@@ -10,8 +10,9 @@ interactive: false
...
     syslog: syslog
+
```
Comment (redhatrises) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_error_action.var:18]: Added empty space here

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var [hunk 5]
```diff
@@ -10,9 +10,10 @@ interactive: false
 
 options:
     default: single
-    email: email
     exec: exec
     halt: halt
     single: single
     suspend: suspend
     syslog: syslog
+    warn_once: warn_once
```
Comment (redhatrises) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var:18]: I don't think a warn_once option should be supported. The recommendation should be syslog.

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_disk_full_action.var [hunk 6]
```diff
@@ -10,9 +10,10 @@ interactive: false
 
 options:
     default: single
-    email: email
```
Comment (redhatrises) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_disk_full_action.var:13]: Why is email removed?

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var [hunk 7]
```diff
@@ -10,9 +10,10 @@ interactive: false
 
 options:
     default: single
-    email: email
```
Comment (redhatrises) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var:13]: Why is email removed?

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_error_action.var [hunk 8]
```diff
@@ -10,8 +10,9 @@ interactive: false
 
 options:
     default: single
-    email: email
```
Comment (redhatrises) [inline linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_error_action.var:13]: Why is email removed?

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var (hunk 3 above)
Comment (vojtapolasek) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_disk_full_action.var:13]: > Why is email removed? Also added empty space here.

Sorry for white spaces, I will fix them all. Email is not mentioned in the man page so I removed it.

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var (hunk 5 above)
Comment (vojtapolasek) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var:18]: > I don't think a warn_once option should be supported. The recommendation should be syslog.

I am just following the latest audisp-remote.conf man page

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_disk_full_action.var (hunk 6 above)
Comment (vojtapolasek) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_disk_full_action.var:13]: > Why is email removed?

It is not mentioned in the list of actions in the man page.

**Comment by vojtapolasek** [comment]
All cases of "email" removal were done because this option disappeared from the man page. But I don't know when it happened.

File: linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var (hunk 5 above)
Comment (redhatrises) [reply linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var:18]: @vojtapolasek remember that we need to provide sane security options not always all the options that a man page provides.

**Pull request merged**